pkg/crc/oc/oc_cache.go

```diff
@@ -13,6 +13,8 @@ import (
 	"path/filepath"
 	"regexp"
 	"strings"
+
+	log "github.com/code-ready/crc/pkg/crc/logging"
```

Comment:
llet's be consistent... we either use `log` or `logging`... or if this conflicts, `crclog` as alias.